package.json

```diff
@@ -96,6 +96,7 @@
     "@typescript-eslint/parser": "^5.59.11",
     "@urql/exchange-auth": "^2.1.6",
     "@vitejs/plugin-react": "^3.1.0",
+    "@vitest/coverage-v8": "^2.1.8",
```

Comment:
To avoid issues or conflicts, we should always make sure that all `vitest` related package versions are equal.

- `vitest`
- `@vitest/coverage-v8`

Reply:
Fixed!